### PR TITLE
slack-config/sig-release: Update teams

### PR DESCRIPTION
**Pull request by justaugustus**
- Add Lauri to teams
- Update Release Manager membership (ref: https://github.com/kubernetes/sig-release/issues/1226, https://github.com/kubernetes/sig-release/issues/1227)
- Rotate Release Team Leads (ref: https://kubernetes.slack.com/archives/C2C40FMNF/p1600212320332200?thread_ts=1600212056.329700&cid=C2C40FMNF)
- Add sig-release-leads group

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

/assign @tpepper @alejandrox1 @saschagrunert @LappleApple @mrbobbytables 
cc: @kubernetes/release-team-leads @kubernetes/release-engineering

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
/lgtm
Thanks!

**Comment by saschagrunert** [comment]
Ah
```
LappleApple: "U011C07244F" is not a valid slack user ID
```

**Review by xmudrii — APPROVED** [review]
/lgtm

**Comment by xmudrii** [comment]
/hold
because of the test failure to avoid retests

**Comment by justaugustus** [comment]
I found a bug in tempelis due to the length of @LappleApple's member ID.

I'm pretty sure it's here:
https://github.com/kubernetes-sigs/slack-infra/blob/1eb97f5f2eb25163f26aba3e9ac6894b6a7c0133/tempelis/config/parser.go#L208-L209

I can push a fix for that, but for now I've left her listed on teams, but commented out in the PR so we don't fail presubmits.

**Review by hasheddan — APPROVED** [review]
/lgtm

**Comment by alejandrox1** [comment]
should i put the approve and then come back to add Lauri or do you want to get the tempelis fix in first?

**Comment by justaugustus** [comment]
@alejandrox1 -- Fixed here: https://github.com/kubernetes/test-infra/pull/19240
This could be approved and once the presubmit updates, it should merge fine.

**Comment by justaugustus** [comment]
/retest
(now that https://github.com/kubernetes/test-infra/pull/19240 has merged.)

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**
